### PR TITLE
[CHK-3172] remove xpay vpos refs

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Remove XPAY VPOS refs

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -7,7 +7,7 @@ REDIS_INSIGHT_PORT=8001
 NODO_MOCK_COMMIT_SHA=master
 NODO_MOCK_PORT=3002
 
-ECOMMERCE_TRANSACTIONS_COMMIT_SHA=main
+ECOMMERCE_TRANSACTIONS_COMMIT_SHA=CHK-3096-remove-xpay-vpos-references
```
Comment (pietro-tota) [inline .env:10]:
```suggestion
ECOMMERCE_TRANSACTIONS_COMMIT_SHA=main
```

File: .env [hunk 2]
```diff
@@ -21,7 +21,7 @@ PGS_PORT=8094
 
 WALLET_PORT=8095
 
-ECOMMERCE_EVENT_DISPATCHER_COMMIT_SHA=main
+ECOMMERCE_EVENT_DISPATCHER_COMMIT_SHA=CHK-3136-delete-xpay-vpos-refs
```
Comment (pietro-tota) [inline .env:24]:
```suggestion
ECOMMERCE_EVENT_DISPATCHER_COMMIT_SHA=main
```

File: pgs-mock/Dockerfile.pgs-mock [hunk 3]
```diff
@@ -1,9 +0,0 @@
-FROM node:18.19.0-alpine
```
Comment (pietro-tota) [inline pgs-mock/Dockerfile.pgs-mock:1]: delete `pagopa-pgs-mock` from docker-compose.yaml file too

File: .env (hunk 1 above)
Comment (giovanniberti) [inline .env:10]: TODO change back to `main`

File: .env (hunk 2 above)
Comment (giovanniberti) [inline .env:24]: TODO change back to `main`

**Review by giovanniberti — DISMISSED** [review]

File: pgs-mock/Dockerfile.pgs-mock (hunk 3 above)
Comment (pietro-tota) [inline pgs-mock/Dockerfile.pgs-mock:1]: delete pagopa-pgs-mock from docker-compose.yaml file too, as for now docker compose build is broken for missing still referenced Dockerfile

**Review by pietro-tota — CHANGES_REQUESTED** [review]
Please remove pgs mock from docker-compose-yaml file

**Review by pietro-tota — APPROVED** [review]

**Review by petretiandrea — APPROVED** [review]

**Pull request merged**